### ISSUE TITLE
Support for react 18

### ISSUE THREAD

**Issue by AnshuBansal6** [jaredpalmer/razzle#1924]
## ❓Question

When should we expect a support for react 18? Any approximate date for razzle next version or else I need to move my project to some other framework

**Comment by aforty** [comment]
What doesn't work? I've been using react@18 for a while now.

**Comment by AnshuBansal6** [comment]
@aforty Are you using razzle ssr with react 18 ?

**Comment by aforty** [comment]
Ah SSR, gotcha. No I'm not.

**Comment by dmitryshelomanov** [comment]
@AnshuBansal6 yes, it works with react 18 https://github.com/jaredpalmer/razzle/issues/1922
